lib/collector/serverless.js

```diff
@@ -301,3 +295,4 @@ class ServerlessCollector {
 }
 
 module.exports = ServerlessCollector
+/* eslint-disable camelcase */
```

Comment:
Is this supposed to be at the bottom of the file?

Reply:
nevermind, I see the same comment earlier in the file.

Reply:
good catch, this should re-enable it